cypress.config.js

```diff
@@ -14,9 +14,19 @@ module.exports = defineConfig({
         generateUser() {
           const randomNumber = Math.ceil(Math.random(1000) * 1000);
           return {
-            username: faker.name.firstName() + `${randomNumber}`,
+            username: faker.person.firstName() + `${randomNumber}`,
             email: 'test' + `${randomNumber}` + '@mail.com',
-            password: '12345Qwert!'
+            password: '12345Qwert!',
+            bio: faker.lorem.sentence()
+          };
+        },
+        generateUser2() {
+          const randomNumber = Math.ceil(Math.random(1000) * 1000);
+          return {
+            username: faker.person.firstName() + `${randomNumber}`,
+            email: 'test' + `${randomNumber}` + '@mail.com',
+            password: '12345Qwert!',
+            bio: faker.lorem.sentence()
           };
         },
```

Comment:
The `generateUser2` function is identical to `generateUser`. If there is no specific reason for having two separate functions, consider removing one to avoid redundancy.